x-pack/filebeat/module/squid/log/test/access1.log-expected.json

```diff
@@ -0,0 +1,5610 @@
+[
+    {
+        "@timestamp": "2006-09-08T04:21:52.000Z",
+        "destination.as.number": 36752,
+        "destination.as.organization.name": "Oath Holdings Inc.",
+        "destination.geo.continent_name": "North America",
+        "destination.geo.country_iso_code": "US",
+        "destination.geo.location.lat": 37.751,
+        "destination.geo.location.lon": -97.822,
+        "destination.ip": [
+            "209.73.177.115"
+        ],
+        "event.action": "TCP_MISS",
+        "event.code": "CONNECT",
+        "event.dataset": "squid.log",
+        "event.module": "squid",
+        "event.original": "1157689312.049   5006 10.105.21.199 TCP_MISS/200 19763 CONNECT login.yahoo.com:443 badeyek DIRECT/209.73.177.115 -",
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.offset": 0,
+        "observer.product": "Proxy",
+        "observer.type": "Proxies",
+        "observer.vendor": "Squid",
+        "related.ip": [
+            "10.105.21.199",
+            "209.73.177.115"
+        ],
+        "rsa.internal.hcode": "DIRECT",
+        "rsa.internal.messageid": "CONNECT",
+        "rsa.investigations.ec_subject": "NetworkComm",
+        "rsa.investigations.ec_theme": "ALM",
+        "rsa.misc.action": [
+            "CONNECT",
+            "TCP_MISS"
+        ],
+        "rsa.misc.content_type": "-",
+        "rsa.misc.result_code": "200",
+        "rsa.network.domain": "login.yahoo.com",
+        "rsa.time.duration_time": 5006,
+        "rsa.time.event_time": "2006-09-08T04:21:52.000Z",
+        "rsa.time.event_time_str": "1157689312",
```

Comment:
seems odd that the event_time_str field is populated with a number.

Reply:
Yep, that's how this parser is defined. squid uses UNIX timestamps, which is captured it in event_time_str (raw). That is used to populate event_time and `@timestamp`.